target/linux/mvebu/cortexa9_neon/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+
+[ ! -e /etc/config/wireless ] && exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+linksys,wrt1200ac|\
+linksys,wrt1900ac-v2|\
+linksys,wrt1900acs|\
+linksys,wrt32x)
+	SKU=$(strings /dev/mtd3|sed -ne 's/^cert_region=//p')
+	WIFIMAC2G=$(macaddr_add $(cat /sys/class/net/eth0/address) +1)
+	WIFIMAC5G=$(macaddr_add $WIFIMAC2G +1)
+	case "$SKU" in
+		AP)
+			REGD=CN
+		;;
+		AU)
+			REGD=AU
+		;;
+		CA)
+			REGD=CA
+		;;
+		EU)
+			REGD=DE
+		;;
+		US)
+			REGD=US
+		;;
+	esac
+
+	WIFIMAC0=$WIFIMAC5G
+	WIFIMAC1=$WIFIMAC2G
```

Comment:
Since the case is gone, this reassignment actually isn't needed anymore either, and one could just use WIFIMAC2G/WIFIMAC5G in the uci commands below directly.